metrics/go_metrics.go

```diff
@@ -781,3 +873,142 @@ func (me *Metrics) RecordAdsCertReq(success bool) {
 func (me *Metrics) RecordAdsCertSignTime(adsCertSignTime time.Duration) {
 	me.adsCertSignTimer.Update(adsCertSignTime)
 }
+
+func (me *Metrics) RecordModuleDuration(labels ModuleLabels, duration time.Duration) {
+	mm, err := me.getModuleMetric(labels)
+	if err != nil {
+		return
+	}
+
+	// Module metrics
+	mm.DurationTimer.Update(duration)
+	// Account-Module metrics
```

Comment:
Please add a line break before the Account-Module comment throughout to improve readability.